.storybook/preview.js

```diff
@@ -25,6 +27,29 @@ function StrictModeToggle({isStrict = false, children}) {
   return <Wrapper>{children}</Wrapper>;
 }
 
+function AppProviderWithKnobs({newDesignLanguage, colorScheme, children}) {
+  const colors = Object.entries(DefaultThemeColors).reduce(
+    (accumulator, [key, value]) => ({
```

Comment:
I'm not a fan of creating a new object on ever loop, can we add to the existing one insead?

```js
    (accumulator, [key, value]) => {
      accumulator[key] = strToHex(color(key, value, 'Theme'));
      return accumulator;
    },
```

Reply:
Maybe, depends on how good your reason is 😄